### PR TITLE
Bump deps in packages/**/package.json

### PR DESCRIPTION
**Pull request by acdlite**

### CODE REVIEW

File: packages/jest-mock-scheduler/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://reactjs.org/",
   "peerDependencies": {
     "jest": "^23.0.1",
-    "scheduler": "^0.11.0"
```
Comment (bvaughn) [inline packages/jest-mock-scheduler/package.json]: Should we really bump this one?

Comment (bvaughn) [reply packages/jest-mock-scheduler/package.json]: Was it intentional that we didn't raise it past 0.11 previously? Or was that a mistake?

Comment (acdlite) [reply packages/jest-mock-scheduler/package.json]: The API changed in 0.14 so yeah. It's private anyway so doesn't really matter.

**Comment by acdlite** [comment]
I know sizebot I'm fixing it!

**Pull request merged**